### DISCUSSION TITLE
Is there a new release planned any time soon?

### DISCUSSION THREAD

**Discussion by ChrisFraun** [jertel/elastalert2#748] [Q&A] [answered]
Since some useful things happened in the last weeks/months, it would be great to have a new release containing these changes.
Is something planned in this direction?
If yes, when?

For me personally only the helm chart release is important for now.

Best,
Christoph

**Comment by jertel** [comment, marked as answer]
Releases are typically 1-2 months apart. The last release was in January. This next release is taking a little longer due to some uncertainties around ES8. I would expect something in the next 1-2 weeks.

**Comment by ChrisFraun** [comment]
Perfect, thanks for the Info!
Looking forward to it!
